temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
```

Comment:
Import the `json` and `logging` modules at the beginning of the file. [important]